desc/equilibrium.py

```diff
@@ -265,6 +270,106 @@ def change_resolution(
         if N_grid is not None and N_grid != self.N_grid:
             self._N_grid = N_grid
 
+    @classmethod
+    def from_near_axis(
+        cls, eq, r=0.1, L=None, M=8, N=None, ntheta=None, spectral_indexing="ansi"
```

Comment:
could we rename `eq` here to something like near_axis_eq or `naeq` or something? we use `eq` a ton of other places to refer specifically to DESC equilibrium objects